### PR TITLE
[#1781] fix package import for org.junit.jupiter.api

### PR DESCRIPTION
**Pull request by cdietrich**
[#1781] fix package import for org.junit.jupiter.api
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.testing/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -28,6 +28,7 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.xtend.lib
 Import-Package: org.apache.log4j;version="1.2.15",
  org.apache.log4j.spi;version="1.2.15",
+ org.junit.jupiter.api;version="[5.0.0,6.0.0)";resolution:=optional,
```
Comment (szarnekow) [inline org.eclipse.xtext.testing/META-INF/MANIFEST.MF:31]: Interesting find. Why did it even compile without the import?

**Review by szarnekow — APPROVED** [review]
Thank you for investigating this!

File: org.eclipse.xtext.testing/META-INF/MANIFEST.MF (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.testing/META-INF/MANIFEST.MF:31]: Cause gradle does not check manifest

Comment (szarnekow) [reply org.eclipse.xtext.testing/META-INF/MANIFEST.MF:31]: Of course... why would we compile an OSGI module with a tools that understands OSGI manifests ...

**Comment by LorenzoBettini** [comment]
As @szarnekow pointed out (https://github.com/eclipse/xtext-core/pull/1782/files#r757780619) "Of course... why would we compile an OSGI module with a tools that understands OSGI manifests ..."
I'd still like to understand why we (Xtext) keep on using (read "banging our head against") Gradle instead of doing everything with the good ol' Maven, which still works like a charm and does not break things with each new release (or if it does, it's quickly fixed or it's easy to fix things).
In particular, from what I understand, none of the committers are really confident with Gradle (at least, that's the impression I had -- I seem to recall that once we completely bricked the metadata on Maven Central... something that I don't even think it's possible when releasing with Maven instead of Gradle).

**Pull request merged**